sql/core/src/test/scala/org/apache/spark/sql/streaming/StreamingJoinSuite.scala

```diff
@@ -136,7 +136,7 @@ abstract class StreamingJoinSuite
     (leftInput, rightInput, select)
   }
 
-  protected def setupWindowedJoinWithRangeCondition(joinType: String)
+  protected def setupJoinWithRangeCondition(joinType: String)
```

Comment:
Fixing nits alone in a PR is not encouraged in general. They can be fixed together when you touch this codes or review other PRs to fix. It's okay since the PR is already open but let's avoid next time.

Reply:
@HyukjinKwon - thanks for pointing it out. Just for my own understanding to better follow next time, is `[MINOR][FOLLOWUP]` PR discouraged, or `[MINOR][FOLLOWUP]` nits (renaming, fixing code style, etc) PR discouraged?

And what's could be a MINOR PR besides nits (maybe refactoring)? Thanks.

Reply:
That's not just a nit, as it brings actual confusion. I actually requested a new PR as I don't like to mix this up with the origin PR, #30395

Reply:
Ah, okay. Gotya.